internal/nametransform/names.go

```diff
@@ -53,6 +56,10 @@ func New(e *eme.EMECipher, longNames bool, raw64 bool) *NameTransform {
 // DecryptName decrypts a base64-encoded encrypted filename "cipherName" using the
 // initialization vector "iv".
 func (n *NameTransform) DecryptName(cipherName string, iv []byte) (string, error) {
+  if n.BypassRegex != nil && n.BypassRegex.MatchString(cipherName) {
```

Comment:
This should only trigger if decrypting the name fails. In other words, try to decrypt first, then check if Bypass matches. I suggest to create a new helper function, `(n *NameTransform) decryptName(cipherName string, iv []byte)` that tries to do the regular decryption. `DecryptName` then only calls `decryptName` and check Bypass when `decryptName` fails.